Composer/packages/client/src/components/TriggerCreationModal/ResolveTriggerWidget.tsx

```diff
@@ -1,13 +1,14 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
please revert the name changes of file, it's a render function not a component

Reply:
Hi @hatpick , I'm using `useState` inside this function, eslint will throw this error.
![image](https://user-images.githubusercontent.com/49866537/120753367-a5301200-c53d-11eb-9ef4-9d56bf92236b.png)

Reply:
I moved useState to parent components, resolved this error.